packages/js/src/plugins/auctionHouseModule/updateAuctionHouse.ts

```diff
@@ -149,4 +159,50 @@ export const updateAuctionHouseBuilder = (
         key: params.instructionKey ?? 'updateAuctionHouse',
       })
     );
+
+  if (!auctionHouse.hasAuctioneer && params.auctioneerAuthority) {
+    const auctioneerAuthority = params.auctioneerAuthority
+      ?.publicKey as PublicKey;
+    const ahAuctioneerPda = findAuctioneerPda(
+      auctionHouse.address,
+      auctioneerAuthority
+    );
+
+    const ahAuctioneerAccount = await metaplex
+      .rpc()
+      .getAccount(ahAuctioneerPda);
```

Comment:
I'm not sure we need this right? If `auctionHouse.hasAuctioneer` is `false` then that means the PDA should have never been created.

Reply:
I've checked it. Actually, PDA always exists, so need to check if the account was initialized.